docs/_static/css/dark.css

```diff
@@ -0,0 +1,622 @@
+/* links */
```

Comment:
Does this file have a license?

Reply:
Some of it is my own work, most was generated using `pygments` via the method explained here: https://github.com/richleland/pygments-css

`pygments` uses [BSD-2](https://github.com/pygments/pygments/blob/master/LICENSE) but it doesn't directly contain this file, just the set of colors that would produce the file. I can't find a license on monokai specifically.